### DISCUSSION TITLE
Request support for [A5-14-0A] and [A5-08-01]

### DISCUSSION THREAD

**Discussion by holger-tangermann** [mak-gitdev/HA_enoceanmqtt#31] [Request] [answered]
May I ask, if these 2 EEP profiles ([A5-14-0A] and [A5-08-01]) could be added. Description for both can be found in the EEPViewer (http://tools.enocean-alliance.org/EEPViewer/#)

Happy to help with testing them.

**Comment by mak-gitdev** [comment]
Hi @holger-tangermann,

Please have a look at the [support-new-eeps](https://github.com/mak-gitdev/HA_enoceanmqtt/tree/support-new-eeps) branch.
I have an initial support for A5-08-01.
Just follow the instructions in the [readme](https://github.com/mak-gitdev/HA_enoceanmqtt/blob/support-new-eeps/README.md) to replace the mapping.yaml file.

Please report then if everything is ok so it could be added to the next release. Thanks

**Reply by holger-tangermann** [reply]
Hi @mak-gitdev , 

thanks for the answer. I have successfully updated the mapping.yaml and tested the EEP A5-08-01. 

These are the entities created in HA.
![image](https://user-images.githubusercontent.com/76705846/211154734-e7a05530-bdf2-45c9-be2c-cbe6172c9ff7.png)
Entities working and providing correct values (checked against a different smart home system):
- ..._lux (brightness)
- ..._pir (motion detection)
- ..._voltage (voltage of internal power supply)

Entities working with reasonable values, which I unforunately cannot check:
- ..._delete
- ..._rssi

Entities, which I cannot check, because they are not provided by my device. 
..._tempC 
..._occ 

So for me my device works as expected. Thanks a lot for realizing it. 

Is there also a possibility to ceate an initial support for the other EEP [A5-14-0A]?

**Comment by mak-gitdev** [comment, marked as answer]
`delete` and `rssi` are special entities generated by HA_enoceanmqtt. You don't have to test them.
Strange that `tempC` and `occ` are not generated by your device. It seems that all A5-08-01 should generate these values.
What is your device reference ? It could be an issue in the added support for A5-08-01.

Regarding A5-14-0A, it will require a new release so you just have to wait a little bit :blush:

**Reply by holger-tangermann** [reply]
>  It seems that all A5-08-01 should generate these values. What is your device reference ? It could be an issue in the added support for A5-08-01.

I do not think it is an issue with the added support. My device is an Eltako FABH65S (https://www.enocean-alliance.org/product/wireless-outdoor-motionbrightness-sensor-fabh65s-wg/). Eltako themselve write in the manual, that they "use A5-08-01 with exceptions". 
![image](https://user-images.githubusercontent.com/76705846/211157323-a0e18d9c-0879-419c-98e6-37f164f67e6f.png)

**Reply by mak-gitdev** [reply]
Ok I see. Eltako devices ...

**Reply by chma73** [reply]
Hello @mak-gitdev  could you finish the A5-14-0A integration? It's seems that open, close, alarm is working - but the window status tilted is not shown.

**Reply by mak-gitdev** [reply]
Hi @chma73, 

This has been completed long tile ago.

There are two entities for the window states:
- `states_2p` which reports only open/closed states
- `states_3p` which reports open/closed/tilted states

So I supposed you are using the `states_2p` entity instead of the `states_3p`.

If you are using the `states_3p` entity and it still don't report the `tilted` state, then please report your log with debug enabled.

**Comment by chma73** [comment]
Hi @mak-gitdev,
I asked because it was not marked in the supported devices
![Ohne Titel](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/137674516/c58fff0f-20e7-40ff-9262-4d91086d62a5)
now I see it with state 3 it works perfect thank you.

**Reply by mak-gitdev** [reply]
@chma73, 

Perfect.

In the supported devices list, an unchecked box means that the EEP is supported but not tested.
I can now check A5-14-0A as you confirmed it works.

Please let me know if you have other working devices not checked in the list.

**Reply by chma73** [reply]
@mak-gitdev ah thanks for the explanation. wish you a happy new year.

**Reply by mak-gitdev** [reply]
@chma,

Thanks. Happy new year
